docs/tre-admins/auth.md

```diff
@@ -1,55 +1,87 @@
+App registrations (represented by service principals) define the various access permissions to the TRE system. There are a total of four main Applications of interest.
 
-You can create the app registrations needed for the API by running the `/scripts/aad-app-reg.sh` script.
+| AAD Application | Description |
+| ----------- | ----------- |
+| TRE API application | This is the main application and used to authenticate access to the [TRE API](../tre-developers/api.md). |
+| TRE Swagger UI | This is used to authenticate identities who wish to use the Swagger UI |
+| Automation App | This application is created so that you can run the  tests or any CI/CD capability. This is particularly important if your tenant is MFA enabled. |
+| Workspace API | Typically you would have an application securing one or more workspaces that are created by TRE. |
 
-Alternatively, you can create the app registrations manually via the [Azure Portal](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app). The requirements are listed below.
+Some of the applications require **admin consent** to allow them to validate users against the AAD. Check the Microsoft Docs on [Configure the admin consent workflow](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/configure-admin-consent-workflow) on how to request admin consent and handle admin consent requests.
 
-!!! note
-    Additional app registrations are required to run the E2E tests, and also to create workspaces - these are not configured by the `aad-app-reg.sh` script. Find information below on how to set these up.
+You can create these applications manually, but `/scripts/aad-app-reg.sh`  does the heavy lifting for you. Should you wish to create these manually via the [Azure Portal](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app). The requirements are defined [below](#Manual-Deployment).
 
-Some of the applications require **admin consent** to allow them to validate users against the AAD. Check the Microsoft Docs on [Configure the admin consent workflow](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/configure-admin-consent-workflow) on how to request admin consent and handle admin consent requests.
+## App registration script
 
-### App registration script
+### TRE API, Swagger and Automation Account
 
-The `/scripts/aad-app-reg.sh` script automatically sets up the app registrations with the required permissions to run Azure TRE. It will create and configure the two main app registrations: **TRE API** and **TRE Swagger UI**.
+The `/scripts/aad-app-reg.sh` script automatically sets up the app registrations with the required permissions to run Azure TRE. It will create and configure all of the app registrations depending on the parameters. You would typically run this script twice; once for the **TRE API** and once for each **Workspace**.
```

Comment:
nit: Is "run this script twice" accurate? Would this be more accurate:

```
You would typically run this script multiple times; once for the **TRE API** and once for each **Workspace**.
```

Reply:
This is a minor nit-pick, so please ignore for the purposes of this PR.

I've updated this branch with `main`